public/_includes/_head-include.jade

```diff
@@ -34,12 +34,13 @@ meta(itemprop="description" content="#{description}")
 meta(itemprop="image" content="https://angular.io/resources/images/logos/standard/shield-large.png")
 
 link(rel="icon" type="image/x-icon" href="/resources/images/icons/favicon.ico")
-link(rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/angular_material/1.0.0/angular-material.min.css")
```

Comment:
You can leave this line here. Just indent it, and place the `if !ng2` line above it.